miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/StorageLocationRestController.java

```diff
@@ -116,6 +117,18 @@ private StorageLocation getFreezer(long id) {
     return createStack(freezer, height);
   }
 
+  @RequestMapping(value = "/rooms", method = RequestMethod.POST)
+  @ResponseStatus(HttpStatus.CREATED)
+  @ResponseBody
+  public StorageLocationDto addRoom(@RequestParam(name = "alias", required = true) String alias,
+      @RequestParam(name = "identificationBarcode", required = false) String identificationBarcode) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)